.github/workflows/main.yml

```
@@ -95,6 +104,83 @@ jobs:
             binaries.tar
 
       - name: Upload load graphs
+        if: ${{ !cancelled() }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: plots
+          path: |
+            **/plot_*.svg
+
+  build-sv2v:
+    name: Build sv2v
+    runs-on: [self-hosted, Linux, X64]
+    container: ubuntu:jammy
+    env:
+      DEBIAN_FRONTEND: noninteractive
+      GHA_MACHINE_TYPE: "n2-highmem-8"
```

```suggestion
      GHA_MACHINE_TYPE: "n2-standard-4"
```

highmem-8 looks like a bit overkill for this job.